src/osd/ECBackend.cc

```diff
@@ -2091,7 +2091,24 @@ void ECBackend::be_deep_scrub(
     return;
   }
 
-  ECUtil::HashInfoRef hinfo = get_hash_info(poid, false);
+  //copy from get_hash_info. Because we already got xattrs, so it don't need
```

Comment:
I'm having second thoughts on this change.  Does the performance of scrub really matter that much to complicate this code?  If so, we should modify get_hash_info() to take optional xattrs so it can use that if needed and available?